pyproject.toml

```diff
@@ -16,7 +16,13 @@ classifiers = [
 ]
 requires-python = ">=3.10"
 
-dependencies = ["pydantic==1.8.2"]
+dependencies = [
+  "pydantic>=2.4.2", # dolma does not work with very old versions of pydantic
+  "dolma@git+https://github.com/allenai/dolma.git", # Install from git until a 0.9.2 package is released
```

Comment:
We can pin to a specific hash for maximum security. Alternatively, we probably want an issue so we don't forget this.